testing/ef_tests/src/cases/fork_choice.rs

```diff
@@ -62,7 +62,7 @@ pub enum Step<B, A, P> {
 #[derive(Debug, Clone, Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct Meta {
-    description: String,
+    _description: String,
```

Comment:
Probably this one is the failing one (and another one) But tbh I don't understand why the `#[derive(Deserialize)]` is not enough to mark as used

Reply:
Oh right, because Deserialize does indeed not read the field. Never mind :+1: